### PR TITLE
doc: make README description clearer and concise

### PR DESCRIPTION
**Pull request by wla80**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ For additional Plotters, see the [Community Plotters](https://github.com/gonum/p
 
 There is a discussion list on Google Groups: gonum-dev@googlegroups.com.
 
-`gonum/plot` is split into a few packages:
+`gonum/plot` is split into following packages:
```
Comment (sbinet) [inline README.md]: `s/following/the following/`

Comment (wla80) [reply README.md]: hmm... I don't see this is a necessary improvement.

Comment (eaburns) [reply README.md]: "the following"

Comment (kortschak) [reply README.md]: English grammar demands the definite article here before "following".

File: README.md [hunk 2]
```diff
@@ -9,12 +9,12 @@ For additional Plotters, see the [Community Plotters](https://github.com/gonum/p
...
-* The `plotter` package provides a standard set of `Plotter`s which use the primitives provided by the `plot` package for drawing lines, scatter plots, box plots, error bars, etc. to a plot. You do not need to use the `plotter` package to make use of `gonum/plot`, however: see the wiki for a tutorial on making your own custom plotters.
-* The `plotutil` package contains a few routines that allow some common plot types to be made very easily. This package is quite new so it is not as well tested as the others and it is bound to change.
-* The `vg` package provides a generic vector graphics API that sits on top of other vector graphics back-ends such as a custom EPS back-end, draw2d, SVGo, X-Window and gopdf.
+* `plot` provides a simple interface for laying out a plot and primitives for drawing to it.
+* `plotter` provides a standard set of `Plotter`s which use the primitives provided by `plot` for drawing lines, scatter plots, box plots, error bars, etc. to a plot. You do not need to apply `plotter` for the use of `gonum/plot`. In addition, see the wiki for a tutorial on making your own custom plotters.
```
Comment (kortschak) [inline README.md]: I don't think either of the changes to this line are an improvement. Please revert them.

File: README.md [hunk 3]
```diff
@@ -9,12 +9,12 @@ For additional Plotters, see the [Community Plotters](https://github.com/gonum/p
...
+* `plotutil` contains routines that allow some common plot types to be created easily. However, `plotutil` is a new package which is not fully tested and is subject to change.
```
Comment (kortschak) [inline README.md]: I don't think this change is necessary.

**Comment by kortschak** [comment]
This has had no movement for over a year.

**Pull request closed without merging**